### PR TITLE
[kubernetes/kubeadm] fix minor nil issues in kudeadm code

### PR DESCRIPTION
**Pull request by tariq1890**
/kind cleanup

**What this PR does / why we need it**:
Minor code quality improvements

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
no
```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/copycerts/copycerts.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ func getDataFromDisk(cfg *kubeadmapi.InitConfiguration, key []byte) (map[string]
 	secretData := map[string][]byte{}
 	for certName, certPath := range certsToTransfer(cfg) {
 		cert, err := loadAndEncryptCert(certPath, key)
-		if err == nil || (err != nil && os.IsNotExist(err)) {
+		if err == nil || os.IsNotExist(err) {
```
Comment (tariq1890) [inline cmd/kubeadm/app/phases/copycerts/copycerts.go]: If err is nil, it'll short circuit. So there is no need to check for `err != nil`

File: cmd/kubeadm/app/util/pkiutil/pki_helpers.go [hunk 2]
```diff
@@ -480,8 +480,7 @@ func parseCSRPEM(pemCSR []byte) (*x509.CertificateRequest, error) {
 	}
 
 	if block.Type != certutil.CertificateRequestBlockType {
-		var block *pem.Block
-		return nil, errors.Errorf("expected block type %q, but PEM had type %v", certutil.CertificateRequestBlockType, block.Type)
+		return nil, errors.Errorf("expected block type %q, but PEM had type %q", certutil.CertificateRequestBlockType, block.Type)
```
Comment (tariq1890) [inline cmd/kubeadm/app/util/pkiutil/pki_helpers.go]: This is nil-access waiting to happen.

**Review by rosti — APPROVED** [review]
Thanks @tariq1890 !
/approve
/lgtm

**Comment by fabriziopandini** [comment]
/approve
/lgtm
Thanks @tariq1890

**Pull request merged**